src/main/java/org/cyclops/integrateddynamics/core/evaluate/variable/ValueTypeDouble.java

```diff
@@ -125,6 +128,13 @@ public ValueTypeInteger.ValueInteger floor(ValueDouble a) {
         return ValueTypeInteger.ValueInteger.of((int) Math.floor(a.getRawValue()));
     }
 
+    @Override
+    public ValueTypeString.ValueString fuzzy(ValueDouble a) {
+        NumberFormat nf = NumberFormat.getCompactNumberInstance(Locale.US, NumberFormat.Style.SHORT);
+        nf.setMaximumFractionDigits(2);
+        return ValueTypeString.ValueString.of(nf.format(a.getRawValue()));
```

Comment:
Also, since the implementation seems identical across all number types, perhaps we can move this to a `default` interface method.

Reply:
I might need some help with that one - when I attempted to move it into `IValueTypenumber.java`, `getRawValue` isn't defined...